### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by bouthouri** [gunar/slack-delete#7]
It's not working for macOS Mojave, google chrome 75

**Comment by gunar** [comment]
Yeah Slack's UI changed and the library needs to be updated (for all systems).
